File: Java/프로그래머스/0/181873. 특정한 문자를 대문자로 바꾸기/특정한 문자를 대문자로 바꾸기.java
class Solution {
    public String solution(String my_string, String alp) {
        char ch[] = my_string.toCharArray();
        for(int i=0; i<ch.length ;i++){
            if(ch[i]==alp.charAt(0)) ch[i]=Character.toUpperCase(ch[i]);
        }
        String answer = String.valueOf(ch);
        return answer;
    }
}

File: JavaScript/프로그래머스/0/181873. 특정한 문자를 대문자로 바꾸기/특정한 문자를 대문자로 바꾸기.js
function solution(my_string, alp) {
    var answer = '';
    var arr = new Array(my_string.length);
    for(var i=0; i<arr.length;i++){
        if(my_string.charAt(i)==alp){
            arr[i]=alp.toUpperCase();
        }
        else {
            arr[i]=my_string.charAt(i);
        }
    }
    for(var i=0; i<arr.length; i++){
        answer+= arr[i];
    }
    return answer;
}